platform/commonUI/edit/bundle.js

```diff
@@ -165,6 +167,15 @@ define([
                     "implementation": SaveAction,
                     "name": "Save",
                     "description": "Save changes made to these objects.",
+                    "depends": [],
+                    "priority": "mandatory"
+                },
+                {
+                    "key": "save-as",
+                    "category": "conclude-editing",
+                    "implementation": SaveAsAction,
+                    "name": "Save As",
```

Comment:
I'd leave name as just "Save" to avoid introducing changes to the UI (unless there have been prior discussions with @charlesh88 establishing that "Save As" would be preferred in this case.)

Also, think we can remove change to CreateAction if the `key` here is `save` instead of `save-as` - the `appliesTo` criteria are sufficient to distinguish between the two, no need to encode that distinction again in CreateAction.